documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -301,92 +210,16 @@ radix_sort Out-of-Place Example
    3 2 1 5 3 3
    5 3 3 3 2 1
 
-radix_sort_by_key Out-of-Place Example
---------------------------------------
-
-.. code:: cpp
-
-   // possible build and run commands:
-   //    icpx -fsycl radix_sort_by_key.cpp -o radix_sort_by_key -I /path/to/oneDPL/include && ./radix_sort_by_key
 
-   #include <cstdint>
-   #include <iostream>
-   #include <sycl/sycl.hpp>
-
-   #include <oneapi/dpl/experimental/kernel_templates>
-
-   namespace kt = oneapi::dpl::experimental::kt;
-
-   int main()
-   {
-      std::size_t n = 6;
-      sycl::queue q{sycl::gpu_selector_v};
-      sycl::buffer<std::uint32_t> keys{sycl::range<1>(n)};
-      sycl::buffer<std::uint32_t> keys_out{sycl::range<1>(n)};
-      sycl::buffer<char> values{sycl::range<1>(n)};
-      sycl::buffer<char> values_out{sycl::range<1>(n)};
-
-
-      // initialize
-      {
-         sycl::host_accessor k_acc{keys, sycl::write_only};
-         k_acc[0] = 3, k_acc[1] = 2, k_acc[2] = 1, k_acc[3] = 5, k_acc[4] = 3, k_acc[5] = 3;
-
-         sycl::host_accessor v_acc{values, sycl::write_only};
-         v_acc[0] = 'r', v_acc[1] = 'o', v_acc[2] = 's', v_acc[3] = 'd', v_acc[4] = 't', v_acc[5] = 'e';
-      }
-
-      // sort
-      auto e = kt::gpu::esimd::radix_sort_by_key<true, 8>(q, keys, values, keys_out, values_out,
-                                                          kt::kernel_param<96, 64>{}); // (8)
-      e.wait();
-
-      // print
-      {
-         sycl::host_accessor k_acc{keys, sycl::read_only};
-         for(std::size_t i = 0; i < n; ++i)
-               std::cout << k_acc[i] << ' ';
-         std::cout << '\n';
-
-         sycl::host_accessor v_acc{values, sycl::read_only};
-         for(std::size_t i = 0; i < n; ++i)
-               std::cout << v_acc[i] << ' ';
-         std::cout << "\n\n";
-         
-         sycl::host_accessor k_out_acc{keys_out, sycl::read_only};
-         for(std::size_t i = 0; i < n; ++i)
-               std::cout << k_out_acc[i] << ' ';
-         std::cout << '\n';
-
-         sycl::host_accessor v_out_acc{values_out, sycl::read_only};
-         for(std::size_t i = 0; i < n; ++i)
-               std::cout << v_out_acc[i] << ' ';
-         std::cout << '\n';
-      }
-
-      return 0;
-   }
-
-**Output:**
-
-.. code:: none
-
-   3 2 1 5 3 3
-   r o s d t e
-
-   1 2 3 3 3 5
-   s o r t e d
-
-
-.. _memory-requirements:
+.. _keys-memory-requirements:
```

Comment:
Should the labels be globally unique? If yes, I suggest to use the name of the page as the prefix, i.e. `_radix-sort-memory-requirements` etc.

Reply:
I've also applied to radix_sort_by_key.rst page.